desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -293,6 +243,11 @@ public void onRemove(final InputColumn<?> sourceColumn) {
     private static final int DEFAULT_WINDOW_WIDTH = 1000;
     private static final int DEFAULT_WINDOW_HEIGHT = 710;
 
+    private static final int SERVER_CHECK_INTERVAL = 2000; // [ms]
```

Comment:
This is a very aggressive check. I would suggest something in an order of magnitude bigger. Every 2 minutes would be more sensible.